ios/magnet/AppDelegate.m

```diff
@@ -11,6 +11,9 @@
 
 #import "RCTRootView.h"
 
+// Include the project headers for swift code. (<project-name>-Swift.h)
+#import "magnet-Swift.h"
```

Comment:
Because AppDelegate.m is an objective-c file, in order to access swift code (NotificationsHelper.swift) the way of including the header is like that, <project-name>-Swift.h . Yes it's crappy but is the way apple provides:

https://developer.apple.com/library/ios/documentation/Swift/Conceptual/BuildingCocoaApps/MixandMatch.html